.github/workflows/generate-reference-docs.yaml

```diff
@@ -2,100 +2,14 @@ name: Deploy Reference Documentation
 
 on:
   workflow_dispatch:
-    inputs:
-      ref:
-        description: 'Choose the branch/tag/ref to deploy the reference documentation from'
-        required: true
-        default: 'main'
-        type: string
```

Comment:
Let's keep this option available.

Reply:
Since you can always pick the triggering branch, what is the value of keeping it?

Reply:
As discussed offline, ref = branch/tag/ref. I can't exactly remember why we added support for this input but I don't think we should drop it anyway.